lib/SearchAndSort/SearchAndSort.js

```diff
@@ -132,6 +133,7 @@ class SearchAndSort extends React.Component {
     module: PropTypes.shape({ // values specified by the ModulesContext
       displayName: PropTypes.node, // human-readable
     }),
+    namespace: PropTypes.string.isRequired,
```

Comment:
A new required prop is a breaking change. Do you think there's a way to handle this dynamically?

Reply:
@zburke I don't think this is a breaking change because the consumers don't have to pass it down or worry about it. It's handled by the SAS internally.

Reply:
🤦 `withNamespace`. Thanks for setting me straight, @mkuklis.